src/pymovements/dataset/dataset.py

```diff
@@ -379,7 +379,7 @@ def apply(
 
         Use apply for upsampling, downsampling or making the sampling rate constant
         using resample:
-        >>> dataset.apply('resample', resampling_rate=2000)# doctest:+ELLIPSIS
+        >>> dataset.apply('resample', resampling_rate=1000)# doctest:+ELLIPSIS
```

Comment:
sampling rate of ToyDataset is 1000, here nothing happens if we change this to 1000 ?

Reply:
good catch, I thought I had to revert a line from #914 but it was already reverted there and 2000 was the original value.

I reverted this change